GettingStarted.md

```diff
@@ -1,16 +1,16 @@
-Moving to 4.0.0-rc1/rc2/rc3 from 3.x.x:
+Moving to 4.0.0/rc1/rc2/rc3 from 3.x.x:
```

Comment:
I think we should now remove reference to rc's

Reply:
Removed